src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -55,6 +64,17 @@ public ReadPileup(final Locatable loc, final List<GATKRead> reads, final List<In
     }
 
     /**
+     * Helper routine for converting a stratified by sample pileup to a list of pileup elements
+     */
+    private static List<PileupElement> stratifiedPileupToList(final Map<String, ReadPileup> stratifiedPileup) {
+        final List<PileupElement> combined = new ArrayList<>();
```

Comment:
It is not, because `new ArrayList(stratifiedPileup.values())` returns a `List<ReadPileup>`, but we need a `List<PileupElement>`